docs/source/migration_mt_django.rst

```diff
@@ -0,0 +1,559 @@
+
+.. code-block:: python
+
+  class Project(models.Model):
+      name = models.CharField(max_length=255)
+      account = models.ForeignKey(Account, related_name='projects',
+                                  on_delete=models.CASCADE)
+      managers = models.ManyToManyField(Manager, through='ProjectManager')
+
+      class Meta:
+          unique_together = [('account', 'name')]
+
+  class Task(models.Model):
+      name = models.CharField(max_length=255)
+      project = models.ForeignKey(Project, on_delete=models.CASCADE,
+                                  related_name='tasks')
+      account = models.ForeignKey(Account, related_name='tasks',
+                                  on_delete=models.CASCADE)
+
+      class Meta:
+          unique_together = [('account', 'name', 'project')]
+
+Then generate the migration with:
+
+.. code-block:: python
+
+  python manage.py makemigrations
+
+Some ``UNIQUE`` constraints are created by the ORM and you will need to explicitly drop them.
+This is the case for ``OneToOneField`` and ``ManyToMany`` fields.
+
+For these cases you will need to:
+1. Find the constraints
+2. Do a migration to drop them
+3. Re-create constraints including the account\_id field
+
+To find the constraints, connect to your database using ``psql`` and run ``\d+ myapp_projectmanager``
+You will see the ``ManyToMany`` (or ``OneToOneField``) constraint:
+
+.. code-block:: sql
+
+  "myapp_projectmanager" UNIQUE CONSTRAINT myapp_projectman_project_id_manager_id_bc477b48_uniq,
+  btree (project_id, manager_id)
+
+Drop this constraint in a migration:
+
+.. code-block:: python
+
+  from django.db import migrations
+
+  class Migration(migrations.Migration):
+
+    dependencies = [
+      # leave this as it was generated
+    ]
+
+    operations = [
+      migrations.RunSQL("""
+        DROP CONSTRAINT myapp_projectman_project_id_manager_id_bc477b48_uniq;
+      """),
+
+Then change your models to have a ``unique_together`` including the ``account\_id``
+
+.. code-block:: python
+
+  class ProjectManager(models.Model):
+      project = models.ForeignKey(Project, on_delete=models.CASCADE)
+      manager = models.ForeignKey(Manager, on_delete=models.CASCADE)
+      account = models.ForeignKey(Account, on_delete=models.CASCADE)
+
+      class Meta:
+          unique_together=(('account', 'project', 'manager'))
+
+And finally apply the changes by creating a new migration to generate these constraints:
+
+.. code-block:: python
+
+  python manage.py makemigrations
+
+3. Updating the models to use TenantModelMixin and TenantForeignKey
+--------------------------------------------------------------------
+
+Next, we'll use the `django-multitenant <https://github.com/citusdata/django-multitenant>`_ library to add account_id to foreign keys, and make application queries easier later on.
+
+In requirements.txt for your Django application, add
+
+::
+
+  django_multitenant>=2.0.0, <3
```

Comment:
I need to change this version before releasing our app. Actually, I need to merge this PR and afterwards after adding all the additions, I need to make a total check since there will be some additions because of other PR's